src/en/experts-sla.md

```diff
@@ -1,149 +1,257 @@
-Title: Managed solutions under SLA
+Title: Managed solutions
 
-# Managed solutions under SLA
+# Managed solutions
 
-[Juju experts](https://jujucharms.com/experts) may designate a Juju
-model as a managed, supported solution by setting an SLA.
+A Juju Expert is a partner backed by professional support at Canonical Ltd.
+Such a partner deploys and supports an infrastructure based on Juju. We call
```

Comment:
(remove an)

Reply:
Hmm.

Reply:
Hmm? You support infrastructure, deploy infrastructure, manage infrastructure. I don't really think there's a case where you prefix it with an.

Reply:
:ok: